PCbuild/pcbuild.sln

```diff
@@ -165,6 +165,10 @@ Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "zlib-ng", "zlib-ng.vcxproj"
 EndProject
 Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "_remote_debugging", "_remote_debugging.vcxproj", "{4D7C112F-3083-4D9E-9754-9341C14D9B39}"
 EndProject
+Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "_zstd", "_zstd.vcxproj", "{07029B86-F3E9-443E-86FB-78AA6D47FED1}"
+EndProject
+Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "zstd", "zstd.vcxproj", "{8D80E018-6A7D-4B46-A7BE-D9E02A20D473}"
+EndProject
```

Comment:
There should be some project dependency information in here - SLN builds don't use the `ProjectReference` elements from the projects, unfortunately. Easiest way is to open in VS, right-click the project and choose "Project Dependencies" (on `_zstd` to select `zstd` I'd guess, and then on `python` to select `_zstd`)

Reply:
Done: [`b9208b6` (#133366)](https://github.com/python/cpython/pull/133366/commits/b9208b60b429c35482a398a61f973e3dfbd8dc57)